diff -Nur gnome-terminal-2.11.2.orig/configure.in gnome-terminal-2.11.2/configure.in
--- gnome-terminal-2.11.2.orig/configure.in	2005-07-25 11:16:15.000000000 +0200
+++ gnome-terminal-2.11.2/configure.in	2005-08-08 19:46:31.000000000 +0200
@@ -54,6 +54,7 @@
 PKG_CHECK_MODULES(TERM, dnl
  gtk+-2.0 >= $GTK_REQUIRED dnl
  gconf-2.0 >= $GCONF_REQUIRED dnl
+ launchpad-integration
  libglade-2.0 dnl
  libgnomeui-2.0 dnl
  libstartup-notification-1.0 >= $STARTUP_NOTIFICATION_REQUIRED dnl
diff -Nur gnome-terminal-2.11.2.orig/src/terminal-window.c gnome-terminal-2.11.2/src/terminal-window.c
--- gnome-terminal-2.11.2.orig/src/terminal-window.c	2005-08-08 16:19:23.000000000 +0200
+++ gnome-terminal-2.11.2/src/terminal-window.c	2005-08-08 19:45:34.000000000 +0200
@@ -29,6 +29,7 @@
 #include <string.h>
 #include <stdlib.h>
 #include <libgnome/gnome-program.h>
+#include <launchpad-integration.h>
 #include <gtk/gtklabel.h>
 #include <gdk/gdkx.h>
 #include <gdk/gdkkeysyms.h>
@@ -1045,6 +1046,8 @@
                               G_CALLBACK (about_callback), window);
   set_menuitem_text (mi, _("_About"), FALSE);
   menuitem_icon_visibility (mi, menus_have_icons);
+
+  launchpad_integration_add_items (menu, -1, TRUE, TRUE);
   
   terminal_window_reread_profile_list (window);
   
