--- gnome-terminal-2.21.3/configure.in	2007-12-04 00:20:23.000000000 +0000
+++ terminal/configure.in	2007-12-05 16:08:30.000000000 +0000
@@ -35,6 +35,7 @@
 PKG_CHECK_MODULES(TERM, dnl
  gtk+-2.0 >= $GTK_REQUIRED dnl
  gconf-2.0 >= $GCONF_REQUIRED dnl
+ launchpad-integration
  libglade-2.0 dnl
  libgnomeui-2.0 dnl
  libstartup-notification-1.0 >= $STARTUP_NOTIFICATION_REQUIRED dnl
--- gnome-terminal-2.21.3/src/terminal-window.c	2007-11-27 03:38:52.000000000 +0000
+++ terminal/src/terminal-window.c	2007-12-05 16:17:41.000000000 +0000
@@ -29,6 +29,7 @@
 #include <string.h>
 #include <stdlib.h>
 #include <libgnome/gnome-program.h>
+#include <launchpad-integration.h>
 #include <gtk/gtklabel.h>
 #include <gdk/gdkx.h>
 #include <gdk/gdkkeysyms.h>
@@ -1074,7 +1075,9 @@
   mi = append_stock_menuitem (menu, GTK_STOCK_ABOUT, NULL,
                               G_CALLBACK (about_callback), window);
   set_menuitem_text (mi, _("_About"), FALSE);
-  
+
+  launchpad_integration_add_items (menu, -1, TRUE, TRUE);
+
   terminal_window_reread_profile_list (window);
   
   terminal_window_set_menubar_visible (window, TRUE);
