diff -Nur gnome-terminal-2.18.2/configure gnome-terminal-2.18.2.new/configure
--- gnome-terminal-2.18.2/configure	2007-09-18 00:16:05.000000000 +0200
+++ gnome-terminal-2.18.2.new/configure	2007-09-18 11:08:11.000000000 +0200
@@ -24074,12 +24074,15 @@
         pkg_cv_TERM_CFLAGS="$TERM_CFLAGS"
     else
         if test -n "$PKG_CONFIG" && \
-    { (echo "$as_me:$LINENO: \$PKG_CONFIG --exists --print-errors \" gtk+-2.0 >= \$GTK_REQUIRED  gconf-2.0 >= \$GCONF_REQUIRED  libglade-2.0  libgnomeui-2.0  libstartup-notification-1.0 >= \$STARTUP_NOTIFICATION_REQUIRED  vte >= \$VTE_REQUIRED  gnome-vfs-2.0 >= \$GNOMEVFS_REQUIRED\"") >&5
-  ($PKG_CONFIG --exists --print-errors " gtk+-2.0 >= $GTK_REQUIRED  gconf-2.0 >= $GCONF_REQUIRED  libglade-2.0  libgnomeui-2.0  libstartup-notification-1.0 >= $STARTUP_NOTIFICATION_REQUIRED  vte >= $VTE_REQUIRED  gnome-vfs-2.0 >= $GNOMEVFS_REQUIRED") 2>&5
+    { (echo "$as_me:$LINENO: \$PKG_CONFIG --exists --print-errors \" gtk+-2.0 >= \$GTK_REQUIRED  gconf-2.0 >= \$GCONF_REQUIRED  launchpad-integration
+ libglade-2.0  libgnomeui-2.0  libstartup-notification-1.0 >= \$STARTUP_NOTIFICATION_REQUIRED  vte >= \$VTE_REQUIRED  gnome-vfs-2.0 >= \$GNOMEVFS_REQUIRED\"") >&5
+  ($PKG_CONFIG --exists --print-errors " gtk+-2.0 >= $GTK_REQUIRED  gconf-2.0 >= $GCONF_REQUIRED  launchpad-integration
+ libglade-2.0  libgnomeui-2.0  libstartup-notification-1.0 >= $STARTUP_NOTIFICATION_REQUIRED  vte >= $VTE_REQUIRED  gnome-vfs-2.0 >= $GNOMEVFS_REQUIRED") 2>&5
   ac_status=$?
   echo "$as_me:$LINENO: \$? = $ac_status" >&5
   (exit $ac_status); }; then
-  pkg_cv_TERM_CFLAGS=`$PKG_CONFIG --cflags " gtk+-2.0 >= $GTK_REQUIRED  gconf-2.0 >= $GCONF_REQUIRED  libglade-2.0  libgnomeui-2.0  libstartup-notification-1.0 >= $STARTUP_NOTIFICATION_REQUIRED  vte >= $VTE_REQUIRED  gnome-vfs-2.0 >= $GNOMEVFS_REQUIRED" 2>/dev/null`
+  pkg_cv_TERM_CFLAGS=`$PKG_CONFIG --cflags " gtk+-2.0 >= $GTK_REQUIRED  gconf-2.0 >= $GCONF_REQUIRED  launchpad-integration
+ libglade-2.0  libgnomeui-2.0  libstartup-notification-1.0 >= $STARTUP_NOTIFICATION_REQUIRED  vte >= $VTE_REQUIRED  gnome-vfs-2.0 >= $GNOMEVFS_REQUIRED" 2>/dev/null`
 else
   pkg_failed=yes
 fi
@@ -24092,12 +24095,15 @@
         pkg_cv_TERM_LIBS="$TERM_LIBS"
     else
         if test -n "$PKG_CONFIG" && \
-    { (echo "$as_me:$LINENO: \$PKG_CONFIG --exists --print-errors \" gtk+-2.0 >= \$GTK_REQUIRED  gconf-2.0 >= \$GCONF_REQUIRED  libglade-2.0  libgnomeui-2.0  libstartup-notification-1.0 >= \$STARTUP_NOTIFICATION_REQUIRED  vte >= \$VTE_REQUIRED  gnome-vfs-2.0 >= \$GNOMEVFS_REQUIRED\"") >&5
-  ($PKG_CONFIG --exists --print-errors " gtk+-2.0 >= $GTK_REQUIRED  gconf-2.0 >= $GCONF_REQUIRED  libglade-2.0  libgnomeui-2.0  libstartup-notification-1.0 >= $STARTUP_NOTIFICATION_REQUIRED  vte >= $VTE_REQUIRED  gnome-vfs-2.0 >= $GNOMEVFS_REQUIRED") 2>&5
+    { (echo "$as_me:$LINENO: \$PKG_CONFIG --exists --print-errors \" gtk+-2.0 >= \$GTK_REQUIRED  gconf-2.0 >= \$GCONF_REQUIRED  launchpad-integration
+ libglade-2.0  libgnomeui-2.0  libstartup-notification-1.0 >= \$STARTUP_NOTIFICATION_REQUIRED  vte >= \$VTE_REQUIRED  gnome-vfs-2.0 >= \$GNOMEVFS_REQUIRED\"") >&5
+  ($PKG_CONFIG --exists --print-errors " gtk+-2.0 >= $GTK_REQUIRED  gconf-2.0 >= $GCONF_REQUIRED  launchpad-integration
+ libglade-2.0  libgnomeui-2.0  libstartup-notification-1.0 >= $STARTUP_NOTIFICATION_REQUIRED  vte >= $VTE_REQUIRED  gnome-vfs-2.0 >= $GNOMEVFS_REQUIRED") 2>&5
   ac_status=$?
   echo "$as_me:$LINENO: \$? = $ac_status" >&5
   (exit $ac_status); }; then
-  pkg_cv_TERM_LIBS=`$PKG_CONFIG --libs " gtk+-2.0 >= $GTK_REQUIRED  gconf-2.0 >= $GCONF_REQUIRED  libglade-2.0  libgnomeui-2.0  libstartup-notification-1.0 >= $STARTUP_NOTIFICATION_REQUIRED  vte >= $VTE_REQUIRED  gnome-vfs-2.0 >= $GNOMEVFS_REQUIRED" 2>/dev/null`
+  pkg_cv_TERM_LIBS=`$PKG_CONFIG --libs " gtk+-2.0 >= $GTK_REQUIRED  gconf-2.0 >= $GCONF_REQUIRED  launchpad-integration
+ libglade-2.0  libgnomeui-2.0  libstartup-notification-1.0 >= $STARTUP_NOTIFICATION_REQUIRED  vte >= $VTE_REQUIRED  gnome-vfs-2.0 >= $GNOMEVFS_REQUIRED" 2>/dev/null`
 else
   pkg_failed=yes
 fi
@@ -24116,14 +24122,17 @@
         _pkg_short_errors_supported=no
 fi
         if test $_pkg_short_errors_supported = yes; then
-	        TERM_PKG_ERRORS=`$PKG_CONFIG --short-errors --errors-to-stdout --print-errors " gtk+-2.0 >= $GTK_REQUIRED  gconf-2.0 >= $GCONF_REQUIRED  libglade-2.0  libgnomeui-2.0  libstartup-notification-1.0 >= $STARTUP_NOTIFICATION_REQUIRED  vte >= $VTE_REQUIRED  gnome-vfs-2.0 >= $GNOMEVFS_REQUIRED"`
+	        TERM_PKG_ERRORS=`$PKG_CONFIG --short-errors --errors-to-stdout --print-errors " gtk+-2.0 >= $GTK_REQUIRED  gconf-2.0 >= $GCONF_REQUIRED  launchpad-integration
+ libglade-2.0  libgnomeui-2.0  libstartup-notification-1.0 >= $STARTUP_NOTIFICATION_REQUIRED  vte >= $VTE_REQUIRED  gnome-vfs-2.0 >= $GNOMEVFS_REQUIRED"`
         else
-	        TERM_PKG_ERRORS=`$PKG_CONFIG --errors-to-stdout --print-errors " gtk+-2.0 >= $GTK_REQUIRED  gconf-2.0 >= $GCONF_REQUIRED  libglade-2.0  libgnomeui-2.0  libstartup-notification-1.0 >= $STARTUP_NOTIFICATION_REQUIRED  vte >= $VTE_REQUIRED  gnome-vfs-2.0 >= $GNOMEVFS_REQUIRED"`
+	        TERM_PKG_ERRORS=`$PKG_CONFIG --errors-to-stdout --print-errors " gtk+-2.0 >= $GTK_REQUIRED  gconf-2.0 >= $GCONF_REQUIRED  launchpad-integration
+ libglade-2.0  libgnomeui-2.0  libstartup-notification-1.0 >= $STARTUP_NOTIFICATION_REQUIRED  vte >= $VTE_REQUIRED  gnome-vfs-2.0 >= $GNOMEVFS_REQUIRED"`
         fi
 	# Put the nasty error message in config.log where it belongs
 	echo "$TERM_PKG_ERRORS" >&5
 
-	{ { echo "$as_me:$LINENO: error: Package requirements ( gtk+-2.0 >= $GTK_REQUIRED  gconf-2.0 >= $GCONF_REQUIRED  libglade-2.0  libgnomeui-2.0  libstartup-notification-1.0 >= $STARTUP_NOTIFICATION_REQUIRED  vte >= $VTE_REQUIRED  gnome-vfs-2.0 >= $GNOMEVFS_REQUIRED) were not met:
+	{ { echo "$as_me:$LINENO: error: Package requirements ( gtk+-2.0 >= $GTK_REQUIRED  gconf-2.0 >= $GCONF_REQUIRED  launchpad-integration
+ libglade-2.0  libgnomeui-2.0  libstartup-notification-1.0 >= $STARTUP_NOTIFICATION_REQUIRED  vte >= $VTE_REQUIRED  gnome-vfs-2.0 >= $GNOMEVFS_REQUIRED) were not met:
 
 $TERM_PKG_ERRORS
 
@@ -24134,7 +24143,8 @@
 and TERM_LIBS to avoid the need to call pkg-config.
 See the pkg-config man page for more details.
 " >&5
-echo "$as_me: error: Package requirements ( gtk+-2.0 >= $GTK_REQUIRED  gconf-2.0 >= $GCONF_REQUIRED  libglade-2.0  libgnomeui-2.0  libstartup-notification-1.0 >= $STARTUP_NOTIFICATION_REQUIRED  vte >= $VTE_REQUIRED  gnome-vfs-2.0 >= $GNOMEVFS_REQUIRED) were not met:
+echo "$as_me: error: Package requirements ( gtk+-2.0 >= $GTK_REQUIRED  gconf-2.0 >= $GCONF_REQUIRED  launchpad-integration
+ libglade-2.0  libgnomeui-2.0  libstartup-notification-1.0 >= $STARTUP_NOTIFICATION_REQUIRED  vte >= $VTE_REQUIRED  gnome-vfs-2.0 >= $GNOMEVFS_REQUIRED) were not met:
 
 $TERM_PKG_ERRORS
 
